sdccc/src/main/java/com/draeger/medical/sdccc/configuration/DefaultTestSuiteModule.java

```diff
@@ -51,4 +55,17 @@ protected void configure() {
         bind(LocalAddressResolver.class).to(LocalAddressResolverImpl.class).in(Singleton.class);
         bind(ManipulationSerializer.class).to(GsonManipulationSerializer.class).in(Singleton.class);
     }
+
+    /**
+     * Provides a singleton Gson instance configured with a custom OptionalTypeAdapter.
+     * @return A configured, singleton Gson instance.
+     */
+    @Provides
+    @Singleton
+    public Gson provideGson() {
+        return new GsonBuilder()
+                .serializeNulls()
```

Comment:
Why not add this to `com.draeger.medical.sdccc.extension.configuration.GsonInstanceKt#getGson` and remove this `provideGson` method here?

It seems now both are missing something.